android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/bitcoin/TestBitcoinSigning.kt

```diff
@@ -129,6 +130,12 @@ class TestBitcoinSigning {
 
         input.addUtxo(utxo1)
 
+        val plan = UTXOPlanner.plan(input.build(), BITCOIN, Bitcoin.TransactionPlan.parser())
+        assertEquals(55_000, plan.amount)
+        assertEquals(75_000, plan.availableAmount)
+        assertEquals(3740, plan.fee)
+        assertEquals(16260, plan.change)
+
         val output = AnySigner.sign(input.build(), BITCOIN, SigningOutput.parser())
```

Comment:
Could you use the output of the plan as input of sign?  That's how it is used in real life.

Reply:
done